src/Composer/Plugin/PluginEvents.php

```diff
@@ -49,6 +49,16 @@ class PluginEvents
      */
     const PRE_FILE_DOWNLOAD = 'pre-file-download';
 
+    /**
+     * The POST_FILE_DOWNLOAD event occurs after downloading a file
```

Comment:
```suggestion
     * The POST_FILE_DOWNLOAD event occurs after downloading a package dist file
```

As PRE_FILE_DOWNLOAD is actually triggered in other circumstances the distinction is important IMO